readthedocs/core/forms.py

```diff
@@ -22,8 +26,10 @@ class UserProfileForm(forms.ModelForm):
 
     class Meta(object):
         model = UserProfile
-        # Don't allow users edit someone else's user page,
+        # Don't allow users edit someone else's user page
         fields = ['first_name', 'last_name', 'homepage', 'allow_ads']
+        if not settings.USE_PROMOS:
```

Comment:
I think its better to append `allow_ads` instead of deleting from index.
It can be something like
```python
fields = ['first_name', 'last_name', 'homepage']
if settings.USE_PROMOS:
    fields.append('allow_ads')
```